src/components/Home/core-service-item.tsx

```diff
@@ -1,4 +1,6 @@
-import Image, { StaticImageData } from "next/image";
+import React from "react";
```

Comment:
Next.js automatically imports React for you, so you don't need to include this line